packages/ui/src/composables/useDropdown.ts

```diff
@@ -7,6 +7,8 @@ import { useClientOnly } from './useClientOnly'
 export type PlacementPosition = 'top' | 'bottom' | 'left' | 'right'
 export type PlacementAlignment = 'start' | 'end' | 'center'
 export type Placement = PlacementPosition | 'auto' | `${PlacementPosition}-${PlacementAlignment}`
```

Comment:
Hi, @xiongmao86! Isn't this type does the feature you want to implement?

Reply:
@rustem-nasyrov, probably not, for example, 'top-left' wouldn't be generated because `left` is not `PlacementAlignment`.